### PR TITLE
impr: simplify gas details, change some costs, enable db compression

### PR DESCRIPTION
**Pull request by jorgemmsilva**

### CODE REVIEW

File: packages/solo/ledgerl1l2.go [hunk 1]
```diff
@@ -156,11 +156,11 @@ type foundryParams struct {
 
 // CreateFoundryGasBudgetIotas always takes 1000 iotas as gas budget and assets for the call
```
Comment (dessaya) [inline packages/solo/ledgerl1l2.go]:
```suggestion
// CreateFoundryGasBudgetIotas always takes 100000 iotas as gas budget and assets for the call
```

File: packages/vm/gas/policy.go [hunk 2]
```diff
@@ -67,14 +66,13 @@ func (p *GasFeePolicy) AffordableGasBudgetFromAvailableTokens(availableTokens ui
...
 	return nominalUnitsOfGas * p.GasNominalUnit
-
 }
 
 func DefaultGasFeePolicy() *GasFeePolicy {
 	return &GasFeePolicy{
 		GasFeeTokenID:          nil, // default is iotas
-		GasNominalUnit:         100, // gas is burned in 100-s and not less than 100
-		GasPricePerNominalUnit: 100, // default is 1 gas == 1 iota
+		GasNominalUnit:         100, // gas is burned in 1000-s and not less than 1000
```
Comment (dessaya) [inline packages/vm/gas/policy.go]: the value of the constant and the comment are not consistent?

**Review by dessaya — APPROVED** [review]

File: packages/vm/gas/policy.go (hunk 2 above)
Comment (lunfardo314) [reply packages/vm/gas/policy.go]: comment is not consistent with the value

Comment (jorgemmsilva) [reply packages/vm/gas/policy.go]: yes, thanks for pointing that out. updated

**Pull request merged**